x-pack/plugins/apm/public/components/app/service_overview/service_overview_charts/service_oveview_mobile_charts.tsx

```diff
@@ -51,6 +62,22 @@ export function ServiceOverviewMobileCharts({
 
   return (
     <EuiFlexGroup direction="column" gutterSize="s">
+      <EuiFlexItem>
+        <MobileFilters
+          serviceName={serviceName}
```

Comment:
Nit: We can get these from url params inside `MobileFilters`

Reply:
I'm going to get it from `useApmServiceContext`